pkgs/top-level/all-packages.nix

```diff
@@ -14958,7 +14958,9 @@ with pkgs;
 
   smtube = libsForQt5.callPackage ../applications/video/smtube { };
 
-  softmaker-office = callPackage ../applications/office/softmaker/softmaker_office.nix { };
+  softmaker-office-nx = callPackage ../applications/office/softmaker/softmaker-office-nx.nix { };
```

Comment:
```
 Attribute `pkgs.softmaker-office-nx` is a new top-level package using `pkgs.callPackage ./pkgs/applications/office/softmaker/softmaker-office-nx.nix { /* ... */ }`.
  Please define it in pkgs/by-name/so/softmaker-office-nx/package.nix instead.
```

Reply:
Yes, I asked in matrix. I'm not sure what to do here.
As the derivation depends on the shared `generic.nix`, it seems weird to split everything apart.
But maybe I missed a smarter design.

Reply:
there is a workaround that is something like `inherit ( import genetic.nix ) foo1 foo2 `

Reply:
sorry i meant 

```
inherit (callPackage ../applications/office/softmaker/softmaker-office-nx.nix { })  softmaker-office-nx ;
```

Reply:
Check out https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#recommendation-for-new-packages-with-multiple-versions